compiler/rustc_middle/src/ty/context.rs

```diff
@@ -4,6 +4,7 @@
 
 pub mod tls;
 
+use hir::def::Namespace;
```

Comment:
should probably merge this import with `use rustc_hir::def::DefKind;` down some dozens of lines.